cdk-infra/lib/stacks/auto-builder-stack.ts

```diff
@@ -21,6 +22,7 @@ export class AutoBuilderStack extends Stack {
     super(scope, id, props);
 
     const queues = new AutoBuilderQueuesConstruct(this, 'queues');
+    const { vpc } = new AutoBuilderVpcStack(this, 'enhancedVpc');
```

Comment:
What is the difference between this VPC stack and the one made in `cdk-infra/bin/cdk-infra.ts`? Why do we need VPC stacks with 2 different IDs?

Reply:
Thanks for pointing this out. I sort of added this without taking into consideration how it's actually being deployed. This whole stack can be deleted.